src/lib/viewers/controls/annotations/AnnotationsControls.scss

```diff
@@ -9,10 +9,3 @@
 .bp-AnnotationsControls-exitBtn {
     display: none;
 }
-
-.ba-Layer--popup,
-.ba-Layer--highlight,
-.ba-Layer--drawing,
-.ba-point-annotation-marker {
```

Comment:
Is this ba-point-annotation-marker class actually used anywhere? I didn't see this in box-annotations.

Reply:
Yeah, I also couldn't find it in annotations, but this needs to move above the text layer to be clickable (it's a fix for expiring embed links - since it's prefixed with `ba` I figured that annotations would be the best place to put it)

Reply:
It's used in a legacy version of `box-annotations` that's only used by the Expiring Embed flow. So, unless we want to release a patch to v2 of `box-annotations`, this specific style will need to stay in `box-content-preview`.